targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/PeripheralPins.c

```diff
@@ -185,7 +185,7 @@ const PinMap PinMap_SPI_SCLK[] = {
     {NC   , NC   , 0}
 };
 
-const PinMap PinMap_SPI_MOSI[] = {
+const PinMap PinMap_SPI_SOUT[] = {
```

Comment:
Why was this changed from MOSI to SOUT? Same for MISO to SIN below.

Reply:
This is because on Freescale's targets the pins aren't MISO & MOSI. If you switch from master to slave, the direction of the pin will not be changed to match it's new role.
On Freescale's targets, the SIN is always the input what ever the mode (master or slave) of the peripheral.